router/src/main.rs

```diff
@@ -543,6 +527,74 @@ pub async fn get_tokenizer_config(api_repo: &ApiRepo) -> Option<HubTokenizerConf
     Some(tokenizer_config)
 }
 
+/// Create a post_processor for the LlamaTokenizer
+pub fn create_post_processor(
+    tokenizer: &Tokenizer,
+    tokenizer_config: &HubTokenizerConfig,
+) -> Option<TemplateProcessing> {
```

Comment:
Why not return an Result and remove all those unwrap() into `?`?

Reply:
great point! updated in latest